### ISSUE TITLE
GENERATE_HTMLHELP=YES with GENERATE_LATEX=YES and \msc comments adds the .EPS files to index.hhp

### ISSUE THREAD

**Issue by XiaoP76** [doxygen/doxygen#10758]
**Describe the bug**
If I use \msc comments in C++ with both
```
GENERATE_HTMLHELP=YES
GENERATE_LATEX=YES
```
the .EPS file names generated by MSC for Latex are falsely added also into the Html Help index.hhp file:

[FILES]
msc_inline_mscgraph_1.png
**_inline_mscgraph_1.eps_**

Because the .EPS files don't exist in the html directory, this results in the Doxygen error message:
`error: failed to run html help compiler on index.hhp`

When run manually, the hhc compiler produces this error message:
```
d:\bug1.9.2>hhc html\index.hhp
Microsoft HTML Help Compiler 4.74.8702
Compiling d:\bug1.9.2\html\KCASystem.chm
HHC5003: Error: Compilation failed while compiling inline_mscgraph_1.eps.
The following files were not compiled:
inline_mscgraph_1.eps
Unable to open html\index.hhp.
```

**To Reproduce**
Extract the attached zip file [bug1.9.2.zip](https://github.com/doxygen/doxygen/files/14663052/bug1.9.2.zip)
and run
`doxygen Manual.doxygen`

**Expected behavior**
The EPS file names from the Latex generation process must not be included in the Html Help index.hhp file.

**Version**
I was able to reproduce the bug from version 1.10 down to version 1.8.11 from 2015.
It most likely occurs in even older versions.
It doesn't occur in version 1.5.9, which I'm updating from.

**Comment by albert-github** [comment]
For this there was already a proposed pull request #8770, I merged it again against the master.

**Comment by XiaoP76** [comment]
I just checked and this is also fixed in 1.11.0, . Thank you !

**Comment by XiaoP76** [comment]
Fixed in 1.11.0.
